components/Button/NavButton.tsx

```diff
@@ -0,0 +1,26 @@
+"use client";
+import Button from "@/components/Button/Button";
+import { ButtonType } from "@/components/Button/utils";
+import { useRouter } from "next/navigation";
+
+const NavButton = ({
+  className,
+  buttonType,
+  link,
+}: {
+  className?: string;
+  buttonType?: ButtonType;
+  link?: string;
+}) => {
+  const router = useRouter();
+  return (
+    <Button
+      label="Get started"
+      buttonType={buttonType || ButtonType.NAVIGATION}
+      className={className}
+      onClick={() => router.push(link || "/account")}
```

Comment:
handle all routes in separate file in separate class.
For now it is just magic string and it will bring many issues in future when routing system become more complex